src/common/com/intellij/plugins/haxe/lang/psi/impl/HaxePsiFieldImpl.java

```diff
@@ -158,6 +158,17 @@ public PsiType getType() {
     return psiType != null ? psiType : HaxePsiTypeAdapter.DYNAMIC;
   }
 
+  @Override
+  @Nullable
+  public HaxeTypeTag getTypeTag() {
+    return null;
```

Comment:
This is the default implementation, everything implemented in child classes. It was done to make PsiField generic, because most of child classes have such methods (excluding enum values fro example)